engine/src/main/java/com/datatorrent/stram/engine/Node.java

```diff
@@ -293,6 +312,7 @@ public void shutdown()
       alive = false;
     }
 
+    /*
```

Comment:
As discussed, this code sets the non-volatile field in the operator thread and therefore acts a backup when above assignment does not materialize in the cache. So please keep it here and add a comment as to why this seemingly redundant code is needed.